ci-operator/step-registry/openshift/e2e/azure/csi/openshift-e2e-azure-csi-workflow.yaml

```diff
@@ -16,12 +13,10 @@ workflow:
     - chain: gather-core-dump
     - chain: ipi-azure-post
     env:
-      # for storage-conf-feature-gate
-      FEATURESET: TechPreviewNoUpgrade
       # for storage-conf-wait-for-csi-driver
       CLUSTERCSIDRIVER: disk.csi.azure.com
       TRUECONDITIONS: AzureDiskDriverControllerServiceControllerAvailable AzureDiskDriverNodeServiceControllerAvailable
```

Comment:
I think these two vars can be removed too.

Reply:
done